build-user-interfaces-by-composing-css-utility-classes-with-tailwind/README.md

```diff
@@ -0,0 +1,39 @@
+# Build User Interfaces by Composing CSS Utility Classes with Tailwind
+
+![Build User Interfaces by Composing CSS Utility Classes with Tailwind](https://user-images.githubusercontent.com/5633704/93652072-b9e56d80-f9e1-11ea-9c80-08e960f3aa49.png)
```

Comment:
This course image is gigantic could you center and resize it
**Example:**
`<p align="center"><img src="https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/450/full/EGH_AWS-TS.png" width="340"></p>`